netebpfext/net_ebpf_ext.c

```diff
@@ -643,6 +662,26 @@ net_ebpf_ext_uninitialize_ndis_handles()
     }
 }
 
+NTSTATUS
+net_ebpf_extension_open_wfp_engine_handle(HANDLE* wfp_engine_handle)
+{
+    if (!wfp_engine_handle) {
+        return STATUS_INVALID_PARAMETER;
+    }
+
+    return FwpmEngineOpen(NULL, RPC_C_AUTHN_WINNT, NULL, NULL, wfp_engine_handle);
+}
+
+NTSTATUS
+net_ebpf_extension_close_wfp_engine_handle(HANDLE wfp_engine_handle)
```

Comment:
Add SAL `_In_` because HANDLE is technically a pointer. There are many precedences of `_In_ Handle` elsewhere in the project.